src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx

```diff
@@ -51,6 +51,7 @@ redirects:
   - /docs/insights/new-relic-insights/understanding-insights/new-relic-insights
   - /docs/insights/using-insights-ui/getting-started/introduction-new-relic-insights
   - /docs/insights/new-relic-insights/decorating-events/insights-attributes
+  - docs/new-relic-solutions/new-relic-one/core-concepts/transition-new-relic-one-insights/
```

Comment:
/ before the docs/ here I think, also I'm not sure but I think we can remove the last / but I don't know if it matters or not

Reply:
done! thanks!